libs/sdk-backend-spi/src/common/settings.ts

```diff
@@ -196,6 +194,19 @@ export interface ISettings {
      */
     enableAxisLabelFormat?: boolean;
 
+    insecureJavascriptEventsEnabled?: boolean;
+    hidePixelPerfectExperience?: boolean;
+    // and rest of existing FFs need to be added too after investigation whether used in KD
```

Comment:
Is this comment still relevant?

Reply:
no it was for whitelisting. Final solution uses blacklisting